src/components/PageWrapper/page-wrapper.css

```diff
@@ -0,0 +1,32 @@
+.green-bg {
+  background: linear-gradient(
+    128.9deg,
+    #76dd78 -18.28%,
+    rgba(118, 221, 120, 0.15) 63.69%
+  );
+  height: 100vh;
```

Comment:
I think you may want to change this height property into "min-height," otherwise you end up with blank space below if the list is long:
<img width="946" alt="pagewrapperbug2" src="https://user-images.githubusercontent.com/82783679/155808153-69080166-0d82-4d1b-935a-255576ecdf7c.png">

Reply:
Also, like my other comment, feel free to disregard if the items are gonna ultimately live in a scrollable div :)